main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/NickGowdy/deveui-cli/client"
+	"github.com/NickGowdy/deveui-cli/device"
+	"github.com/NickGowdy/deveui-cli/processor"
+	"github.com/joho/godotenv"
+)
+
+/*
+deveui-cli is a Go CLI program.
+It is used for concurrently generating unique 16-character (hex) identifier called a DevEUI.
+These are generated by the program and registered via an external (LoRaWAN) API.
+
+Usage:
+
+	go run main.go (locally)
+
+	go run deveui-cli (docker)
+
+Once this program starts, it will listen to syscall.SIGTERM, syscall.SIGINT via a channel.
+This is to handle any unexpected terminations of the program and to resume processing DevEUIs.
+*/
+func main() {
+	if err := godotenv.Load(".env"); err != nil {
+		panic("error loading.env file")
+	}
+	godotenv.Load(".env")
+	baseurl := os.Getenv("BASE_URL")
+
+	maxConcurrentJobs, err := strconv.Atoi(os.Getenv("MAX_CONCURRENT_JOBS"))
```

Comment:
This set of env var loading/parsing gets in the way of reading what `main()` is doing, it could make sense to extract these out into another function that returns a struct containing all the configuration values.